src/raylib/model/Animation.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Animation
+*/
+
+#ifndef RAYLIB_MODEL_ANIMATION_HPP_
+#define RAYLIB_MODEL_ANIMATION_HPP_
+
+extern "C"
+{
+#include <raylib.h>
+}
+
+#include <string>
+#include "Model.hpp"
+
+namespace raylib
+{
+    namespace model
+    {
+        /// The Animation class
+        class Animation {
+          public:
+            /// Animation constructor. Used to initialize an animation from a file
+            /// File format accepted: obj and probably some other
+            ///
+            /// @param animPath the relative path to the animation to import
+            Animation(const std::string &animPath);
```

Comment:
`std::string_view`

Reply:
no because I need the c_str() function

Reply:
or I could use
`std::string(animPath.data()).c_str()` later IDK what is the best